server/src/integrations/integrations.module.ts

```diff
@@ -16,23 +16,16 @@ import { assert } from 'src/utils/assert';
 const S3StorageModuleFactory = async (
   environmentService: EnvironmentService,
 ): Promise<S3StorageModuleOptions> => {
-  const fileSystem = environmentService.getStorageType();
-  const bucketName = environmentService.getStorageLocation();
-  const region = environmentService.getStorageRegion();
-
-  if (fileSystem === 'local') {
-    return { bucketName };
-  }
-
-  assert(region, 'S3 region is not defined');
+  const bucketName = environmentService.getStorageS3Name();
+  const region = environmentService.getStorageS3Region();
 
   return {
-    bucketName,
+    bucketName: bucketName ?? '',
```

Comment:
I'm only moving the problem to here. I believe the S3Module should not be instanciated if S3 is not configured in env variables. I would encapsulated LocalDriver and S3Driver behind one common FileStorage Interface.
To be discussed and treated in another PR!

Reply:
Should we track this as a new task ?

Reply:
Yes, I'll discuss it with Jeremy tomorrow and create a task based on discussion!

Reply:
Created here: https://github.com/twentyhq/twenty/issues/510 we can discuss it tomorrow